provider/pkg/provider/remote/copyController.go

```diff
@@ -177,10 +177,8 @@ func sftpCopy(sftp *sftp.Client, sourcePath, destPath string) error {
 
 		if !strings.HasSuffix(sourcePath, "/") {
 			dest = filepath.Join(dest, filepath.Base(sourcePath))
-			err = sftp.Mkdir(dest)
-			if err != nil {
-				return fmt.Errorf("failed to create remote directory %s: %w", dest, err)
-			}
+			// It's ok if the dir exists, we'll copy into it.
+			_ = sftp.Mkdir(dest)
```

Comment:
Instead of ignoring the error, can we check if its a "directory already exists" error? It's probably going to wind up as [this code](https://github.com/pkg/sftp/blob/c8fe1f69640c3d92b05e1d7f0072addd6ece3ed2/sftp.go#L54).

Reply:
Good point. I went for a clearer approach instead and `stat` the remote first.